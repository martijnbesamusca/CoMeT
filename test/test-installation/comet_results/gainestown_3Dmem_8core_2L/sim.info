{'cmdline': ['/home/martijn/Documents/uva/thesis/CoMeT/run-sniper',
             '-v',
             '-s',
             'memTherm_core',
             '-n',
             '4',
             '-c',
             'gainestown_3Dmem_8core_2L',
             '--',
             '/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation/test'],
 'git_revision': '70a94ea0a46a09195fc7cc97e7411ec90733f6a3',
 'host': '114c1c70b953',
 'pin_version': 'unknown',
 'rusage': (24.867396,
            1.611389,
            108672,
            0,
            0,
            0,
            52570,
            0,
            0,
            0,
            4600,
            0,
            0,
            0,
            260080,
            234),
 'snipercmd': ['bash',
               '-c',
               '/home/martijn/Documents/uva/thesis/CoMeT/lib/sniper -c /home/martijn/Documents/uva/thesis/CoMeT/config/base.cfg --general/total_cores=4 --general/output_dir=/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation --config=/home/martijn/Documents/uva/thesis/CoMeT/config/nehalem.cfg --config=/home/martijn/Documents/uva/thesis/CoMeT/config/gainestown_3Dmem_8core_2L.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation/sim.scripts.py -g --hooks/script0args= -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpRC03i9/run_benchmarks'],
 't_elapsed': 29.65772795677185,
 't_start': 1653910916.861351,
 'tracecmds': [['/home/martijn/Documents/uva/thesis/CoMeT/record-trace',
                '-o',
                '/tmp/tmpRC03i9/run_benchmarks',
                '-v',
                '--pa',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                '/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation/test']],
 'user': None,
 'vmem': 791560192L}
