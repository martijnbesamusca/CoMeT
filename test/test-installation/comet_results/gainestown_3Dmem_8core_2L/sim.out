                                     | Core 0     | Core 1     | Core 2     | Core 3     | Core 4     | Core 5     | Core 6     | Core 7    
  Instructions                       |   21246393 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  Cycles                             |    9695520 |    9695520 |    9695520 |    9695520 |    9695520 |    9695520 |    9695520 |    9695520
  IPC                                |       2.19 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00
  Time (ns)                          |    2693200 |    2693200 |    2693200 |    2693200 |    2693200 |    2693200 |    2693200 |    2693200
  Idle time (ns)                     |        -69 |    2693200 |    2693200 |    2693200 |    2693200 |    2693200 |    2693200 |    2693200
  Idle time (%)                      |      -0.0% |     100.0% |     100.0% |     100.0% |     100.0% |     100.0% |     100.0% |     100.0%
Branch predictor stats               |            |            |            |            |            |            |            |           
  num correct                        |    2597545 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  num incorrect                      |      26960 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  misprediction rate                 |      1.03% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00%
  mpki                               |       1.27 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00
TLB Summary                          |            |            |            |            |            |            |            |           
  I-TLB                              |            |            |            |            |            |            |            |           
    num accesses                     |    2890060 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    num misses                       |         46 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    miss rate                        |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00
  D-TLB                              |            |            |            |            |            |            |            |           
    num accesses                     |    7876047 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    num misses                       |        903 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    miss rate                        |      0.01% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.04 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00
  L2 TLB                             |            |            |            |            |            |            |            |           
    num accesses                     |        949 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    num misses                       |        405 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    miss rate                        |     42.68% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.02 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00
Cache Summary                        |            |            |            |            |            |            |            |           
  Cache L1-I                         |            |            |            |            |            |            |            |           
    num cache accesses               |    2890060 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    num cache misses                 |        930 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    miss rate                        |      0.03% |       inf% |       inf% |       inf% |       inf% |       inf% |       inf% |       inf%
    mpki                             |       0.04 |        inf |        inf |        inf |        inf |        inf |        inf |        inf
  Cache L1-D                         |            |            |            |            |            |            |            |           
    num cache accesses               |    7876047 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    num cache misses                 |      49523 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    miss rate                        |      0.63% |       inf% |       inf% |       inf% |       inf% |       inf% |       inf% |       inf%
    mpki                             |       2.33 |        inf |        inf |        inf |        inf |        inf |        inf |        inf
  Cache L2                           |            |            |            |            |            |            |            |           
    num cache accesses               |      50938 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    num cache misses                 |      48986 |          0 |          0 |          0 |          0 |          0 |          0 |          0
    miss rate                        |     96.17% |       inf% |       inf% |       inf% |       inf% |       inf% |       inf% |       inf%
    mpki                             |       2.31 |        inf |        inf |        inf |        inf |        inf |        inf |        inf
DRAM summary                         |            |            |            |            |            |            |            |           
  num dram accesses                  |      10180 |      10195 |      10665 |      10167 |      10171 |      10122 |      10119 |      10142
  average dram access latency (ns)   |      37.48 |      38.06 |      48.70 |      37.46 |      37.49 |      38.04 |      37.46 |      37.46
  average dram queueing delay        |       0.05 |       0.63 |      11.27 |       0.03 |       0.06 |       0.61 |       0.03 |       0.03
  average dram bandwidth utilization |      3.00% |      3.00% |      3.00% |      3.00% |      3.00% |      3.00% |      3.00% |      3.00%
Coherency Traffic                    |            |            |            |            |            |            |            |           
  num loads from dram                |      16312 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  num loads from dram cache          |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  num loads from remote cache        |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0
